src/com/sap/piper/JenkinsUtils.groovy

```diff
@@ -19,6 +19,31 @@ static boolean hasTestFailures(build){
     return action && action.getFailCount() != 0
 }
 
+boolean addWarningsNGParser(String id, String name, String regex, String script, String example = ''){
+    def classLoader = this.getClass().getClassLoader()
+    def config = classLoader.loadClass('io.jenkins.plugins.analysis.warnings.groovy.ParserConfiguration', true)?.getInstance()
```

Comment:
[naming] config is in the context of the piper steps an occupied term: the config map as produced by the config helper. Maybe it would be better to choose another name here in order to avoid confusion.